home/views.py

```diff
@@ -33,7 +34,8 @@ def is_volunteer(user):
 def index(request):
     if request.user.is_authenticated:
         return redirect('home:dashboard')
-    return render(request, 'home/index.html')
+    initiatives=Initiative.objects.all()    
+    return render(request, 'home/index.html',{'initiatives':initiatives})
```

Comment:
Use proper formatting (spaces around `=` while assigning something, space after `,` and `:`, etc.)